prow/github/types.go

```diff
@@ -160,6 +160,13 @@ type User struct {
 	HTMLURL string `json:"html_url"`
 }
 
+// Collaborator represents a GitHub repository collaborator
+type Collaborator struct {
```

Comment:
Is there a reason we cannot add Permissions to the User struct (or do not want to)?

Reply:
It just looked like a better option to keep `User` and `Collaborator` separate, to better match the GitHub API which has different structs for them, though with mostly common fields:
[Get a user](https://developer.github.com/v3/users/?#get-a-single-user)
[List Collaborators](https://developer.github.com/v3/repos/collaborators/#list-collaborators)

Please let me know if they need to be combined as User.

Reply:
Those look like the same type `"type": "User"` but with different fields included. Adding the missing missing fields to User will avoid the need to adapt between classes and this approach -- add missing fields as they become necessary -- is the general pattern we take with the client.

Reply:
Thank you, I have updated the PR.